build.gradle.kts

```diff
@@ -8,7 +8,7 @@ plugins {
 }
 
 group = "com.hibiscusmc"
-version = "2.7.3-DEV"
+version = "2.7.3"
```

Comment:
Small but could you put this on the right version (2.7.3-DEV)?

Reply:
doneso